target/snitch_cluster/sw/snax/gemmx/src/snax-gemmx-lib.c

```diff
@@ -21,19 +21,17 @@ int32_t gen_subtraction_config(int8_t subtraction_a, int8_t subtraction_b) {
 }
 
 int32_t gen_csr0_config(uint8_t input_zp_i, uint8_t output_zp_i,
-                        uint8_t shift_i, uint8_t max_int_i) {
+                        uint8_t max_int_i, uint8_t min_int_i) {
     // encode the configuration into a single 32-bit integer
-    return ((int32_t)max_int_i << 24) | ((int32_t)shift_i << 16) |
+    return ((int32_t)min_int_i << 24) | ((int32_t)max_int_i << 16) |
            ((int32_t)output_zp_i << 8) | (int32_t)input_zp_i;
 }
 
```

Comment:
does this work for negative integers? I think the sign bits will then overflow into each other

Reply:
It works for negative integers as the test indicates.
The min_int_i is always -127, the max_int_i  is 128.
input_zp_i and output_zp_i can be negative integers.